32 updates to ssf.txt

PL1-Arcade opened the pull request:
Tested all games with values over 525.
41.23% checked. (1206 of 2925)

PL1-Arcade committed:
47 updates to ssf.txt

Tested all games with values over 550.
40.10% checked. (1173 of 2925)
Stats include all changes currently in the main repo Develop branch.

PL1-Arcade committed:
Merge branch 'Jakobud:develop' into develop

PL1-Arcade committed:
32 updates to ssf.txt

Tested all games with values over 525.
41.23% checked. (1206 of 2925)

PL1-Arcade commented:
Not sure why this PR auto-included the already approved "47 updates" PR entry -- didn't see a way to prevent it.  Synced my develop branch to yours before doing this edit and PR so it should've recognized that the "47 updates" PR was already incorporated.  :man_shrugging: 

Also not sure when the "1942" and "1943kai" duplicate lines appeared.  Skimmed the rest of the file looking for more duplicate entries, but didn't find any.

Endprodukt commented:
You might want to include these: 

calspeed,870
gticlub,1106
midnrun,620
thrilld,700
windheat,580

PL1-Arcade commented:
Thanks for the 5 updates, Endprodukt.  :smiley: 

I'll include them in my next PR along with a note acknowledging your contributions. 

Scott

PL1-Arcade committed:
37 updates to ssf.txt

Tested all games with values over 500.
42.68% checked. (1250 of 2929)
Endprodukt provided the new values for calspeed, gticlub, midnrun, thrilld, and windheat.

PL1-Arcade commented:
Endprodukt,

Your updates are included in the latest commit.  https://github.com/Jakobud/skipstartupframes/pull/52/commits/6db09c246aeefa7fcb9b38397e83e709d456ee7f

Thanks again for your contributions.  :smiley: 

Scott
